Products/PleiadesEntity/skins/PleiadesEntity/place_view.pt

```diff
@@ -118,6 +118,15 @@
         <p tal:condition="not:portal_state/anonymous">Red titles indicate drafting-state items. Yellow titles indicate pending-state.</p>
 
         <div tal:condition="not:context/@@iterate/baseline|nothing">
+
+          <div id="representativePointField" class="field">
+            <label>Representative Point:</label>
+            <dl tal:replace="structure context/@@representative-point"></dl>
+            <div tal:condition="not:portal_state/anonymous">
```

Comment:
What's this doing here? Also, this whole block should probably be in both this section and the section where baseline is defined.